scripts/library.sh

```diff
@@ -351,11 +352,14 @@ function report_go_test() {
 function start_latest_knative_serving() {
   header "Starting Knative Serving"
   subheader "Installing Istio"
+  echo "Installing Istio CRD from ${KNATIVE_ISTIO_CRD_YAML}"
   kubectl apply -f ${KNATIVE_ISTIO_CRD_YAML} || return 1
+  echo "Installing Istio from ${KNATIVE_ISTIO_YAML}"
   kubectl apply -f ${KNATIVE_ISTIO_YAML} || return 1
   wait_until_pods_running istio-system || return 1
   kubectl label namespace default istio-injection=enabled || return 1
   subheader "Installing Knative Serving"
+  echo "Installing Serving from ${KNATIVE_SERVING_RELEASE}"
```

Comment:
nit-picking: just for consistency between subheader and this message:
```suggestion
   echo "Installing Knative Serving from ${KNATIVE_SERVING_RELEASE}"
```

Reply:
You LGTMed but did not hold it, so the PR was merged without your suggestion. :/

Reply:
yeah it's no big deal, I don't think it should be a PR blocker. We can add it in the future